src/main.rs

```diff
@@ -19,12 +21,37 @@ struct Args {
     /// The OpenOCD RTT server port
     #[arg(default_value_t = 9090)]
     port: u16,
+
+    /// Dump target for OpenOCD RTT data
+    #[arg(default_value = "")]
+    file: String,
+}
+
+/// Format data string
+fn format_data(data:String) -> std::string::String {
+    let now = Local::now().format("%T%.3f");
+    let formatted_string = format!("{} {}", format!("[{}]", now).bright_black(), data);
+    return formatted_string;
+}
```

Comment:
What if we print it as `<time>;<rtt-message>`. This would work for csv parsers as we can define the delimiter as `;` in a csv viewer, while also not cluttering original file for human readability.

Reply:
Yes that might be a solution, although it's specifically meant for smicolon-separated CSV files.
For now, I would probably stick to human readable lines by default, like this:
```
[12:34:56.789] RTT Message
```
And we might think of some flags like:
- `--no-output-timestamps`: to omit timestamps to the output file
- `--output-delimiter <char>`: to separate the timestamp from the message using the user-provided delimiter (default might be `,`), resulting in the following output to file:
```
<timestamp><delimiter><rtt-message>
```